### PR TITLE
Greyscale wirecutters

### PR DESCRIPTION
**Pull request by ghost**
Greyscales wirecutters.

:cl: More Robust Than You
imageadd: Wirecutters can come in even more colors now!
/:cl:

### CODE REVIEW

File: code/game/objects/items/tools/wirecutters.dm [hunk 1]
```diff
@@ -20,14 +21,34 @@
 	toolspeed = 1
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 50, acid = 30)
 	var/random_color = TRUE
+	var/static/list/wirecutter_colors = list(
+		"blue" = rgb(24, 97, 213),
```
Comment (ChangelingRain) [inline code/game/objects/items/tools/wirecutters.dm]: use hex codes, not the rgb proc

File: code/game/objects/items/tools/wirecutters.dm [hunk 2]
```diff
@@ -20,14 +21,34 @@
...
+
 
+/obj/item/wirecutters/Initialize()
+	. = ..()
+	if(random_color) //random colors!
+		icon_state = "cutters"
+		var/our_color = pick(wirecutter_colors)
+		add_atom_colour(wirecutter_colors[our_color], FIXED_COLOUR_PRIORITY)
+		update_icon()
+	if(prob(75))
```
Comment (ChangelingRain) [inline code/game/objects/items/tools/wirecutters.dm]: what's this?

Comment (MoreRobustThanYou) [reply code/game/objects/items/tools/wirecutters.dm]: iirc this was just to spread them out if they're on top of each other

Comment (ChangelingRain) [reply code/game/objects/items/tools/wirecutters.dm]: yes, but why are you adding it?

Comment (MoreRobustThanYou) [reply code/game/objects/items/tools/wirecutters.dm]: dunno, this is 5 month old code

**Comment by ShizCalev** [comment]
+1 to #31871

**Review by ShizCalev — CHANGES_REQUESTED** [review]
Group the cutter icons with the rest please.

**Review by ShizCalev — APPROVED** [review]

**Comment by ninjanomnom** [comment]
\> no images
\> no head art honcho man approval
@ShizCalev with as big a change as greyscaling, why :(

**Comment by MoreRobustThanYou** [comment]
@ninjanomnom well it works

**Comment by ninjanomnom** [comment]
it working doesn't mean shit, a sprite change isn't some mechanical feature

**Pull request merged**